cmd/health.go

```diff
@@ -0,0 +1,94 @@
+package cmd
+
+import (
+	"encoding/json"
+	"github.com/humio/cli/api"
+	"github.com/spf13/cobra"
+	"os"
+)
+
+func newHealthCmd() *cobra.Command {
+	var (
+		textFlag       bool
+		rawFlag       bool
+		versionFlag    bool
+		uptimeFlag     bool
+		failFlag       bool
+		warnAsDownFlag bool
+		selectChecks   []string
+	)
+
+	cmd := &cobra.Command{
+		Use:   "health",
```

Comment:
I am not sure, if "health" the right top level command? Maybe it is.
Maybe we should consider making these kinds of command subcommands to something like `cluster` or something like that, so different areas have different grouping top-level commands.

Reply:
Hmm, `status` is a top-level command. Do you think it would fit under `cluster`?